SPECS/icu/icu.spec

```diff
@@ -29,6 +30,7 @@ It contains the libraries and header files to create applications
 
 %prep
 %setup -q
+%patch 0 -p1
```

Comment:
Could you switch this to `%autosetup`, please? It makes it easier to not accidentally miss applying new patches.